lib/grape/dsl/configuration.rb

```diff
@@ -72,6 +72,21 @@ def desc(description, options = {}, &config_block)
           namespace_setting :description, options
           route_setting :description, options
         end
+
+        def description_field(field, value=nil)
+          if value
+            description = route_setting(:description) || route_setting(:description, {})
```

Comment:
This is a bit hard to read, you're having a side-effect assignment on the right-hand. Do this:

```ruby
description = route_setting(:description)
description ||= route_setting(:description, {}) 
```